src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,57 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+  it(`should return error for valid email and password less 
+  than 8 characters`, () => {
+    const invalidPassword = validateRegisterForm('test@mail.com', 'P@ss1');
```

Comment:
what about boundary values?) 8-16 contains at least 1 digit, 1 special character, and 1 uppercase letter. So use boundary values techniques for these requirements and cover them by automated tests: 7, 8, 16, 17, 10 without capital, 10 without spec char, 10 without digit